exercises/custom-set/custom_set_test.rb

```diff
@@ -3,114 +3,303 @@
 require 'minitest/autorun'
 require_relative 'custom_set'
 
+# rubocop:disable Metrics/LineLength
```

Comment:
Yes, it does make sense to attack this in the common file.  I did not like it when I did it either.  I am not addressing it here though, as it is not the 'source'.  The source is the linked issues, I think I have linked them in the PR.